easybuild/easyconfigs/g/GDRCopy/GDRCopy-2.3-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDRCopy'
+version = '2.3'
+
+homepage = 'https://github.com/NVIDIA/gdrcopy'
+description = "A low-latency GPU memory copy library based on NVIDIA GPUDirect RDMA technology."
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['b85d15901889aa42de6c4a9233792af40dd94543e82abe0439e544c87fd79475']
+
+builddependencies = [
+    ('binutils', '2.39'),
+    ('Autotools', '20220317'),
+    ('pkgconf', '1.9.3'),
+]
+
+prebuildopts = "PATH=$PATH:/sbin "  # ensures that ldconfig is found
+buildopts = "config lib prefix=%(installdir)s"
+
+installopts = 'prefix=%(installdir)s'
+
+github_account = 'NVIDIA'
+# This easyconfig only installs the library of GDRCopy. Please keep in mind
+# that GDRCopy also needs the following kernel modules at runtime:
+#
+# 1. Kernel module for GDRCopy: improves Host to GPU communication
+# https://github.com/NVIDIA/gdrcopy
+# RPM: 'gdrcopy-kmod', DEB: 'gdrdrv-dkms'
+# Requirements: version of GDRCopy kernel module (gdrdrv.ko) >= 2.0
+#
+# 2. (optional) Kernel module for GPUDirect RDMA: improves GPU to GPU communication
+# https://github.com/Mellanox/nv_peer_memory
+# RPM: 'nvidia_peer_memory'
+# Requirements: Mellanox HCA with MLNX_OFED 2.1
+#
+# These kernel modules are not listed as system dependencies to lower the system
+# requirements to build this easyconfig, as they are not needed for the build.
+
+skipsteps = ['configure']
```

Comment:
please move up `skipsteps` above `prebuildopts` (and the large comment along with it)